mathjax3-ts/core/MmlTree/MmlNodes/mtr.ts

```diff
@@ -64,7 +64,19 @@ export class MmlMtr extends AbstractMmlNode {
                     .appendChild(child);
             }
         }
-        super.setChildInheritedAttributes(attributes, display, level, prime);
+        const calign = (this.attributes.get('columnalign') as string)
+                       .replace(/^\s+/, '').replace(/\s+$/, '').split(/ +/);
```

Comment:
> Is that not just a trim and split?

Yes.  I only learned about `trim()` recently, and don't always remember to use it.

> Why not use the SPLIT method?

Because it is in the `mtable` file, not the `mtr` one.  It should probably be moved to `util/strings` instead.